### ISSUE TITLE
specify channel axis in blob converters

### ISSUE THREAD

**Issue by bwanglzu** [jina-ai/jina#3521]
`blob` property are used for images, it's nice to have if we could define which axis to put channel, since

`pytorch` and `paddle` supports shape `Batch, Channel, Height, Width` while `tensorflow` supports `Batch, Height, Width, Channel`.

When doing the conversion, we expect:
```python
d.convert_image_datauri_to_blob(channel_axis=1)  # BCHW, default value since pytorch seems to be very popular
d.convert_image_datauri_to_blob(channel_axis=3)  # BHWC
```

**Comment by bwanglzu** [comment]
close since it already exist called `color_axis`
